jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/composite/BootstrapDropdown.java

```diff
@@ -1,23 +1,20 @@
-package com.epam.jdi.light.ui.bootstrap.elements.complex;
+package com.epam.jdi.light.ui.bootstrap.elements.composite;
 
 import com.epam.jdi.light.common.JDIAction;
-import com.epam.jdi.light.elements.base.UIBaseElement;
 import com.epam.jdi.light.elements.common.UIElement;
-import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.composite.Section;
 import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
-import com.epam.jdi.light.ui.bootstrap.asserts.DropdownAssert;
+import com.epam.jdi.light.ui.bootstrap.asserts.BootstrapDropdownAssert;
 import com.epam.jdi.light.ui.bootstrap.elements.common.Button;
 
-import java.util.Arrays;
-import java.util.List;
+import static com.epam.jdi.light.asserts.core.SoftAssert.assertSoft;
 
-public class Dropdown extends UIBaseElement<DropdownAssert> {
+public class BootstrapDropdown extends Section {
```

Comment:
Our dropdown didn't implement any interfaces and extends regular Section. Don't we need at all methods like select(), list(), etc?

Reply:
BootstrapDropdown is just a section. We have another one class  `DropdownMenu extends BootstrapDropdown` with such methods.

Reply:
Oh, okay